cmd/terramate/cli/cli.go

```diff
@@ -193,21 +213,37 @@ func newCLI(
 		}
 	}
 
+	log.Trace().
+		Str("actionContext", "newCli()").
+		Str("stack", wd).
```

Comment:
Maybe some of the duplication duplication on the fields can be removed by using subloggers, like this:

https://go.dev/play/p/UR0RegwskPW

Docs:

https://github.com/rs/zerolog#sub-loggers-let-you-chain-loggers-with-additional-context

Also, maybe something like "action" or "operation" is more succinct than "actionContext".

Reply:
Good idea thanks!